### ISSUE TITLE
declaration-colon-newline-after for comment containing URL

### ISSUE THREAD

**Issue by thommeo** [stylelint/stylelint#4069]
> Clearly describe the bug

There is false positive for comment containing URL.

> Which rule, if any, is the bug related to?

`declaration-colon-newline-after`

> What CSS is needed to reproduce the bug?

```html
<style lang="scss" scoped>
.test {
  // http://easings.net/#easeInOutSine
  animation:
    // http://cubic-bezier.com/#0,1,.3,.99
    slow-pulsar-start 1s cubic-bezier(0, 1, 0.3, 0.99) 1,
    // http://cubic-bezier.com/#.44,0,.57,1
    slow-pulsar 4s cubic-bezier(0.44, 0, 0.57, 1) 1s infinite;
}
</style>
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard-scss"
}
```

or

```json
{
  "declaration-colon-newline-after": "always-multi-line"
}
```

> Which version of stylelint are you using?

`10.0.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint src/Test.vue`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Not sure.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The line `// http://easings.net/#easeInOutSine` is not flagged.
The `// http://cubic-bezier.com/#0,1,.3,.99` is flagged for missing new line after colon.
The next comment line `// http://cubic-bezier.com/#.44,0,.57,1` is not flagged.

The following warnings were flagged:

```shell
src/Test.vue
 5:12  ✖  Expected newline after ":" with a multi-line declaration   declaration-colon-newline-after
```

If run with `--fix` here is the result:

```html
<style lang="scss" scoped>
.test {
  // http://easings.net/#easeInOutSine
  animation:
    // http:
//cubic-bezier.com/#0,1,.3,.99
    slow-pulsar-start 1s cubic-bezier(0, 1, 0.3, 0.99) 1,
    // http://cubic-bezier.com/#.44,0,.57,1
    slow-pulsar 4s cubic-bezier(0.44, 0, 0.57, 1) 1s infinite;
}
</style>
```

Changing `// ...` comments to `/* ... */` doesn't help.

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

I can't reproduce the issue. Could you create a repository with a reproducible issue, please?
